### PR TITLE
Adding demo5

### PR DESCRIPTION
**Pull request by bfuzzy1**
Pinged Harry to let him know this is ready for his review.

### CODE REVIEW

File: shell/demo/demo5.ps1 [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="ca9b22be-93d5-4902-95f4-4bc43a817b73"
+$TEST_INTRO="Will your computer quarantine a malicious python file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious python file was used by the malware authors Portugal and Brazil
+during a supply chain attack in February 2023.
```
Comment (harry-prelude) [inline shell/demo/demo5.ps1:20]: is there a more specific name for this attack?

Comment (bfuzzy1) [reply shell/demo/demo5.ps1:20]: "software supply chain attack" is as specific as we can go for this one I think

File: shell/demo/demo5.ps1 [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+This test will attempt to download a defanged malicious file to your disk - containing a signature for the Colour-Blind malware used in a recent supply chain attack - to see how your endpoint defenses will respond.
```
Comment (harry-prelude) [inline shell/demo/demo5.ps1:22]: This test will attempt to download a defanged malicious `python` file

Comment (bfuzzy1) [reply shell/demo/demo5.ps1]: resolved

File: shell/demo/demo5.ps1 [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="ca9b22be-93d5-4902-95f4-4bc43a817b73"
+$TEST_INTRO="Will your computer quarantine a malicious python file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious python file was used by the malware authors Portugal and Brazil
+during a supply chain attack in February 2023.
+
+This test will attempt to download a defanged malicious python file to your disk - containing a signature for the Colour-Blind malware used in a recent supply chain attack campaign - to see how your endpoint defenses will respond.
```
Comment (harry-prelude) [inline shell/demo/demo5.ps1:22]: replace `recent` with `2023`

**Review by harry-prelude — APPROVED** [review]

**Comment by harry-prelude** [comment]
Well done.

**Comment by bfuzzy1** [comment]
> Well done.

Had a great reviewer!

**Review by privateducky — APPROVED** [review]

**Pull request merged**